common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerActivity.java

```diff
@@ -80,6 +87,26 @@ protected void onStop() {
         super.onStop();
     }
 
+    @Override
+    protected void onDestroy() {
+        // If result hasn't been received. It means that this activity is prematurely killed.
+        // In this case, we want to properly cancel the activity so that it doesn't block MSAL from making a subsequent interactive call.
```

Comment:
Don't we broadcast a message that is supposed to cancel the activity already in the broker?

Reply:
We do in the broker process. This is on MSAL side.